src/ProcessManager/ChromeManager.php

```diff
@@ -28,11 +28,13 @@
 
     private $process;
     private $arguments;
+    private $options;
 
-    public function __construct(?string $chromeDriverBinary = null, ?array $arguments = null)
+    public function __construct(?string $chromeDriverBinary = null, ?array $arguments = null, ?array $options = null)
     {
         $this->process = new Process([$chromeDriverBinary ?? $this->findChromeDriverBinary()], null, null, null, null);
         $this->arguments = $arguments ?? $this->getDefaultArguments();
+        $this->options = $options ?? $this->getDefaultOptions();
```

Comment:
Can you use `array_merge` to merge data by the user provided and default values (in case it's not provided). The default value can then be `[]`.

Reply:
Yup! was just copying args